dandi/validate.py

```diff
@@ -46,10 +44,7 @@ def validate_bids(
     import bidsschematools
     from bidsschematools.validator import validate_bids as validate_bids_
 
-    validation_result = validate_bids_(
-        paths,
-        schema_version=schema_version,
-    )
+    validation_result = validate_bids_(paths, exclude_files=["dandiset.yaml"])
```

Comment:
if this is the one incompatibility, in the future for such ones we might even better stay compatible with both versions for a while (e.g. base on . That would allow to instruct users to e.g. downgrade back to prior version of dependency if new version introduced some relevant regression etc.  So please keep that in mind @TheChymera in the future.

Reply:
specifically here actually we could have stayed then compatible with 3.7 and this PR could have concentrated really on bidsschematools compatibility aspect. 
3.7 support Ends in 1 month and 1 week (27 Jun 2023)

ok, this time let's do it, but in the future -- see above.